### PR TITLE
doc: flags of rit init and rit update repo

### PR DESCRIPTION
**Pull request by maurineimirandazup**
How to use flags in rit init and rit update repo commands.

Signed-off-by: maurineimirandazup <maurinei.miranda@zup.com.br>

### CODE REVIEW

**Review by GuillaumeFalourd — APPROVED** [review]
LGTM 👍🏼

File: content/pt-br/Standard Inputs/Comandos Core.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ Estas flags estão disponíveis a partir da versão 2.8 do Ritchie.
...
 
+### Comando de inicialização do Ritchie
+
+rit init
+
+```text
+rit init --sendMetrics="yes" --addCommons="yes" --runType="local"
+```
+
 ### Comandos de Credenciais
```
Comment (jessica-tw) [inline content/pt-br/Standard Inputs/Comandos Core.md:28]:
```suggestion
### Comandos de credenciais
```

File: content/pt-br/Standard Inputs/Comandos Core.md [hunk 2]
```diff
@@ -95,6 +103,12 @@ rit delete repo
 rit delete repo --name=repo_name
 ```
 
+rit update repo
+
+```text
+rit update repo --name="commons" --version="2.0.0"
+```
+
 ### Comandos de Workspace
```
Comment (jessica-tw) [inline content/pt-br/Standard Inputs/Comandos Core.md:112]:
```suggestion
### Comandos de workspace
```

**Review by jessica-tw — COMMENTED** [review]
TW review, done! ;)

**Pull request merged**